hs_core/templates/pages/genericresource.html

```diff
@@ -1494,10 +1494,10 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 }
 
                 // If does not exist, add it
-                if (!exists && keyword[i] != "") {
+                if (!exists && keyword[i] != "" && keyword[i].length <= 100) {
```

Comment:
What is this hard coded "100"? Is it related to the "keyword_maxlength = Subject._meta.get_field('value').max_length" on line 209 in hs_core/views/__init__.py